index.js

```diff
@@ -889,6 +892,14 @@ RingPop.prototype.handleOrProxyAll =
         }
     };
 
+RingPop.prototype.allowJoins = function allowJoins() {
```

Comment:
Are the `allowJoins` and `denyJoins` just helpers for testing ? We should consider leaving a comment about that.

Or is there an application / production use case for these methods ?